oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
So... is this going to be `_pkce.py`, or do you feel that this module should be directly used by users of the `oauth2client` library? @jonparrott and I pretty strongly feel that this module shouldn't be library-public, but we're willing to be persuaded by well-reasoned argument.